docs/apis/open-data.rst

```diff
@@ -26,54 +26,54 @@ Dataset
+	"Related Content", "field_related_content", "", `references <https://project-open-data.github.io/v1.1/schema/#references>`_
+	"", "uuid", `dct:identifier <https://www.w3.org/TR/vocab-dcat/#Property:dataset_identifier>`_, `identifier <https://project-open-data.github.io/v1.1/schema/#identifier>`_
+	"", "modified_date", `dct:modified <https://www.w3.org/TR/vocab-dcat/#Property:dataset_modified_date>`_, `modified <https://project-open-data.github.io/v1.1/schema/#modified>`_
+	"", "release_date", `dct:issued <https://www.w3.org/TR/vocab-dcat/#Property:dataset_release_date>`_, `issued <https://project-open-data.github.io/v1.1/schema/#issued>`_
 
 The following properties are provided by the Open Data Federal Extras module and have no equivilant in DCAT. They are only relevant to U.S. federal agencies.
 
 .. csv-table::
-	:header: "DKAN Field/Property", "POD property"
+	:header: "DKAN Field/Property label", "DKAN Field/Property machine name", "POD property"
 
-	"field_odfe_bureau_code", `bureauCode <https://project-open-data.cio.gov/v1.1/schema/#bureauCode>`_
-	"field_odfe_data_quality", `dataQuality <https://project-open-data.cio.gov/v1.1/schema/#dataQuality>`_
-	"field_odfe_investment_uii", `primaryITInvestmentUII <https://project-open-data.cio.gov/v1.1/schema/#primaryITInvestmentUII>`_
-	"field_odfe_program_code", `programCode <https://project-open-data.cio.gov/v1.1/schema/#programCode>`_
-	"field_odfe_system_of_records", `systemOfRecords <https://project-open-data.cio.gov/v1.1/schema/#systemOfRecords>`_
+	"Bureau Code", "field_odfe_bureau_code", `bureauCode <https://project-open-data.cio.gov/v1.1/schema/#bureauCode>`_
+	"Data Quality", "field_odfe_data_quality", `dataQuality <https://project-open-data.cio.gov/v1.1/schema/#dataQuality>`_
+	"Primary IT Investment UII", "field_odfe_investment_uii", `primaryITInvestmentUII <https://project-open-data.cio.gov/v1.1/schema/#primaryITInvestmentUII>`_
+	"Program Code", "field_odfe_program_code", `programCode <https://project-open-data.cio.gov/v1.1/schema/#programCode>`_
+	"System of Records", "field_odfe_system_of_records", `systemOfRecords <https://project-open-data.cio.gov/v1.1/schema/#systemOfRecords>`_
 
 
 Resource / Distribution
 ***********************
 
 .. csv-table::
-	:header: "DKAN Field/Property", "DCAT-AP property", "POD property"
-
-	"", "dcat:accessURL", "accessURL"
-	"", "dct:conformsTo", "conformsTo"
-	"", "", "describedBy",
-	"", "", "describedByType"
-	"body", "dct:description", "description"
-	"field_link_remote_file || field_upload", "dcat:downloadURL", "downloadURL"
-	"field_format", "", "format"
-	"field_upload:mime", "dcat:mediaType", "mediaType"
-	"title", "title", "dct:title"
+	:header: "DKAN Field/Property label", "DKAN Field/Property machine name", "DCAT-AP property", "POD property"
+
+	"", , "dcat:accessURL", "accessURL"
+	"", , "dct:conformsTo", "conformsTo"
+	"", , "", "describedBy",
```

Comment:
@rhabbachi I think the last comma on 73 is adding an extra column

Reply:
Yep fixed. Good catch!